packages/cli/src/commands/styles/utils/figma.utils.ts

```diff
@@ -572,7 +572,7 @@ const generateTypographyTokens = <T extends NodeDef>(
     typographyMap,
     ''
   ).flatMap((nodeDoc)=>{
-    console.log(nodeDoc);
+    //console.log(nodeDoc);
```

Comment:
Remove the code instead of commenting it here.

Reply:
I didn't add that code there, it came from the main branch, left it commented out in case @rafRangler needs it again

Reply:
ill remove it

Reply:
Done, removed